openreview/venue_request/venue_request.py

```diff
@@ -1122,17 +1122,35 @@ def setup_request_form(self):
                 'required': True,
                 'order': 4
             },
+            'venue_type': {
+                'description': 'Please select the option that best describes the type of venue you\'re hosting. If neither option is appropriate, type your anwser into the text box.',
```

Comment:
If this is a dropdown, then users can only pick from the dropdown and they can't type in their own answer